CRFHeartRate/CRFHeartRate/Info.plist

```diff
@@ -17,7 +17,7 @@
 	<key>CFBundleShortVersionString</key>
 	<string>1.0</string>
 	<key>CFBundleVersion</key>
-	<string>43</string>
+	<string>15</string>
```

Comment:
Curious why this was so much higher than the xcode project's build number.

Reply:
Whoops, thanks